app/pages/Logs.js

```diff
@@ -53,17 +52,18 @@ class Logs extends React.Component {
             <Feed.Event>
               <Feed.Content>
                 <Feed.Summary>
-                  <Label size="mini" color={this.labelColor(entry.type)}>
-                    {capitalize(entry.type)}
+                  <Label color={determineLabelColor(entry.type)} image horizontal>
+                    <Icon name={determineLabelIcon(entry.type)} />
+                    {capitalize(entry.source)}
+                    {entry.name && <Label.Detail>{entry.name}</Label.Detail>}
                   </Label>
-                  {capitalize(entry.source)} {entry.name ? ` - ${entry.name}` : ''} <Feed.Date>{entry.date}</Feed.Date>
+                  <Feed.Date>{entry.date}</Feed.Date>
                 </Feed.Summary>
                 <Feed.Extra>
                   <div dangerouslySetInnerHTML={{ __html: entry.message }} />
                 </Feed.Extra>
               </Feed.Content>
             </Feed.Event>
```

Comment:
```suggestion
            </Feed.Event>
          <Divider />
```

Reply:
@Xzandro you can commit this and merge if you'd like to keep it.

Reply:
Thanks for adding that back. I will leave it in. As you said, these things are a little bit subjective, but it's just more friendly looking in my opinion.